### ISSUE TITLE
Warn users if they use the default of uncompressed csv outputs

### ISSUE THREAD

**Issue by bloodearnest** [opensafely-core/cohort-extractor#897]
Probably with link to documenation.

**Comment by StevenMaude** [comment]
This could well be a useful :adhesive_bandage: to help dissuade people from using plain CSV.

I haven't run enough projects to know how visible would such a warning be, in the midst of other log output? Is it likely to be seen?

Another and/or additional possibility is having the OpenSAFELY CLI more centrally audit a project, and check for antipatterns. At the moment, it's possible to have CSV output from Data Builder, which might cause similar problems in future.

**Comment by bloodearnest** [comment]
Good point, doing it in opensafely-cli is probably better. Moving it there.
